docker/Dockerfile

```diff
@@ -148,6 +183,10 @@ LABEL org.opencontainers.image.created=$BUILD_DATE
 ARG GITREF=unknown
 LABEL org.opencontainers.image.revision=$GITREF
 
+ARG USERID=10003
+ARG GROUPID=10003
+RUN chown -R ${USERID}:${GROUPID} /app
```

Comment:
I had to do this, otherwise I got the "can't write to a readonly database" sqlite error again.